devicetypes/smartthings/zwave-water-valve.src/zwave-water-valve.groovy

```diff
@@ -80,6 +79,7 @@ def parse(String description) {
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.basicv1.BasicReport cmd) {
+
```

Comment:
Not sure that an empty line here is necessary. Let's go ahead and remove it.

Reply:
ok